subworkflows/ingest.nf

```diff
@@ -9,7 +9,7 @@ workflow ingest {
         fastq
         //metadata
     main:
-        get_params_and_versions()
```

Comment:
This might be breaking something somewhere, will have to revisit what versions to get and how, grabbing this from the kraken2-server might be doable?